frontend/src/components/pages/CampOverview/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  AspectRatio,
+  Box,
+  Container,
+  Flex,
+  HStack,
+  Image,
+  Tag,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import placeHolderImage from "../../../assets/germany.jpeg";
+import costIcon from "../../../assets/coin.svg";
+import locationIcon from "../../../assets/location.svg";
+import ageIcon from "../../../assets/person.svg";
+import { Camp } from "../../../types/CampsTypes";
+import CampsAPIClient from "../../../APIClients/CampsAPIClient";
+
+export type CampOverviewProps = {
+  campId: string;
+};
+
+const CampOverview = (): JSX.Element => {
+  const [camp, setCamp] = useState<Camp>();
+  const campId = "62c098e7b4a7a433a7622ff4"; // hardcoded for now, TODO: update this
+
+  // TODO: update the statuses
+  enum Status {
+    PUBLISHED = "Published",
+    DRAFT = "Draft",
+  }
+
+  const status = camp?.active ? Status.PUBLISHED : Status.DRAFT;
+
+  useEffect(() => {
+    const getCampInfo = async () => {
+      const campResponse = await CampsAPIClient.getCampById(campId);
+      setCamp(campResponse);
```

Comment:
We should have some kind of error handling for if campResponse is `null`
```suggestion
      if (campResponse) setCamp(campResponse);
      else // display an error or just let it remain null
```